src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -153,20 +159,47 @@ export default {
 			return 'preview'
 		},
 		previewUrl() {
+			const userId = this.$store.getters.getUserId()
 			if (this.hasTemporaryImageUrl) {
 				return this.localUrl
 			}
 
-			if (this.previewAvailable !== 'yes' || this.$store.getters.getUserId() === null) {
+			if (this.previewAvailable !== 'yes') {
 				return OC.MimeType.getIconUrl(this.mimetype)
 			}
 
+			// max size of a gif for which we allow direct embedding
+			let maxGifSize = 3145728
+			const caps = getCapabilities()
+			if (caps) {
+				maxGifSize = caps.spreed.config.previews['max-gif-size']
+			}
```

Comment:
```suggestion
			const maxGifSize = getCapabilities()?.caps?.spreed?.config?.previews?.['max-gif-size'] || 3145728
```

Reply:
depends what hurts the eyes more :smile:

Reply:
You mean using kebab-case keys? :wink:

Reply:
no? I? mean? the? question? marks? :smile:

Reply:
It's the new fancy optional chaining :heart_eyes: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Love it!